README.md

```diff
@@ -66,6 +66,12 @@ If you want to use the client but are not using [Composer](https://getcomposer.o
 - You can now use the client like above described.
 - If you don't want to have an additional build step, just check the `vendor/` folder into your repository.
 
+Testing
+-------
+
+The Killbill PHP client uses [phpunit](https://phpunit.de/) as testing framework. Tests can either be run locally using `npm test` or against
```

Comment:
Whoops, i'm sorry. This should've actually been `composer test`. No need to install `npm` or `node`.

Too much context switching 🙈